src/components/carousel/DotsPaginationButton.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+interface DotsButtonProps {
+  onClick: () => void;
+  isCurrent: boolean;
+  currentDot: number;
+}
+
+const DotsPaginationButton: FC<DotsButtonProps> = ({
+  onClick,
+  isCurrent,
+  currentDot,
+}) => {
+  const { dotsPaginationButton, dotsPaginationButtonActive } =
+    useMultiStyleConfig('Carousel');
+
+  return (
+    <chakra.button
+      __css={isCurrent ? dotsPaginationButtonActive : dotsPaginationButton}
+      onClick={onClick}
+      aria-current={isCurrent}
+      aria-label={`selected dot ${currentDot}`}
```

Comment:
this label doesn't really make sense. It should describe what it is. Maybe something like
`slide indicator ${currentDotNumber} of ${totalDots}`?